src/contracts/Flyweight.sol

```diff
@@ -32,10 +32,15 @@ contract Flyweight {
     oracleNodeAddress = _oracleNodeAddress;
     ordersCount = 0;
     tokenWhitelist = new TokenWhitelist(
-      uniswapRouterAddress,
+      _uniswapRouterAddress,
```

Comment:
compiled using 0.8.17, which *technically* compiles & references the correct value `uniswapRouterAddress`, even though the state hasn't updated yet on the blockchain.

but did this change anyways... it was making me uncomfy :/